contracts/EuroToken.sol

```diff
@@ -121,8 +126,8 @@ contract EuroToken is
         public
     {
         require(_balances[msg.sender] >= amount);
```

Comment:
then you can drop this. `sub` already checks for overflows

Reply:
Sub does an `assert`, which is not appropriate here. It needs to be `required`.

Reply:
this is what Zeppelin is doing everywhere and I also do not like. But then with this check what is the point of using `sub`? We should not use math functions mindlessly...
I'm going to change Zeppelin token implementations accordingly, look
```
// Check is not needed because sub(_allowance, _value) will already throw if this condition is not met
// require (_value <= _allowance);
```

Reply:
ok.

Reply:
no worries, will be separate PR